src/js/components/Form/stories/TriggerValidationUsingFormField.js

```diff
@@ -38,13 +38,14 @@ export const TriggerValidationUsingFormField = () => (
             },
           ]}
           validateOn="blur"
-        />
-
+        >
+          <TextInput aria-label="blur" id="blur" aria-required name="blur" />
```

Comment:
Shouldn't need `aria-label` given the `htmlFor`

Reply:
done